moesif/plugins/log.lua

```diff
@@ -336,14 +372,16 @@ function _M.log_response(handler)
                     end 
 
                     -- Send events to moesif
-                    local _, _ = handler:httpCall("moesifprod", send_events_headers, payload, 5000, true)
+                    local send_events_headers, _ = handler:httpCall("moesifprod", send_events_headers, payload, 5000, false)
 
-                    if debug then 
-                        handler:logDebug("[moesif] Events sent successfully")
-                    end
+                    if send_events_headers[":status"] ~= nil and tonumber(send_events_headers[":status"]) == 201 then 
```

Comment:
proxy doesn't return 201, it only return 200. Can you extend this check to 200 also?